packages/design-system/src/components/Button/style.scss

```diff
@@ -131,9 +134,25 @@
 
   .btn__loader {
     position: absolute;
-    top: 50%;
-    transform: translate(-50%, -50%);
-    left: 50%;
+    top: 0;
+    left: 0;
+    opacity: 1;
+    width: 100%;
+    height: 100%;
+    padding: 6px 16px;
```

Comment:
Can't you use `top: 50%` + `transform: translateY(-50%);` ? I mean this padding doesn't look good in a component with poissible various sizes (button can be small, normal and large)

Reply:
https://github.com/livechat/design-system/pull/179/files#diff-ce3c4a2a8c708ff0a84602f45be811a5R102 - different padding for smaller button